Makefile

```diff
@@ -97,16 +97,13 @@ benchmarks-basic-build:
 
 benchmarks-basic-run:
 	$(MAKE) benchmarks-basic-build
-	scripts/benchmark-runner.py run \
-		--path $(BENCHMARKS_BASIC_DIR) ${EXTRA_BENCHMARK_FLAGS}
-
-benchmarks-basic-dump:
-	$(MAKE) benchmarks-basic-run EXTRA_BENCHMARK_FLAGS="--dump-path ${BENCHMARKS_DUMP_DIR}"
+	cd scripts/veloxbench/veloxbench && \
+	conbench cpp-micro --run-name="manual" --run-reason="manual"
```

Comment:
This will require someone to install conbench in their environment, yeah? Once we have the adapters work in https://github.com/conbench/conbench/pull/369 we won't need to depend on that though, so I think this is fine for now — but @pedroerp might know more if people use this command locally enough that we shouldn't complicate it for now?

Reply:
Right, this will require `conbench` to be installed and the `.conbench` file set up correctly (for posting results to conbench to work). @pedroerp said on https://github.com/facebookincubator/velox/pull/2193/files#r937186383 that no one was using this workflow before, so I figured it was okay to change the prerequisites.

Reply:
We don't really do anything with the results of circleCI runs, but it would be nice to still support some sort of local run without having a conbench dependency.

Reply:
The main reason I forced the conbench entrypoint in this PR is that in other projects we've found it valuable to record manual runs in conbench for data analysis purposes. It's nice to keep track of those metrics in the database as a side effect of running benchmarks locally.

Having said that, it is a bit of extra work on developers to get the conbench stuff set up. We could do most of this for devs in the Makefile (perhaps using [pipx](https://pypa.github.io/pipx/) to ensure `conbench` is installed in an isolated venv), though they'd still have to set up a `.conbench` file, which we could give detailed instructions for.

If that's too much prescriptivism I'm happy to split the scripts as you suggested!